docs/core/tools/dotnet-install-script.md

```diff
@@ -179,7 +179,6 @@ The install scripts do not update the registry on Windows. They just download th
   Represents a specific build version. The possible values are:
 
   - `latest` - Latest build on the channel (used with the `-Channel` option).
-  - `coherent` - Latest coherent build on the channel; uses the latest stable package combination (used with Branch name `-Channel` options).
```

Comment:
We are dropping the coherent feature. We do not think it works correctly and it has not been a reasonable value for some time. 

Docs folks: Is it OK for us to just drop it, or do we need a note explaining the date on which we dropped it?

Reply:
OK to just drop.